### PR TITLE
Added missing variables for ember-frost-button

### PR DESCRIPTION
**Pull request by job13er**
This is a #feature#

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,42 @@
...
+Modify your `ember-cli-build.js` to include:
+
+
+```javascript
+const frostTheme = require('frostTheme')
+
+const app = new EmberAddon(defaults, {
     sassOptions: {
       includePaths: [
         frostTheme.includePaths
```
Comment (sglanzer-fire) [inline README.md:30]: I can't figure out why, but this isn't working - I've had to fall back to the explicit path 'node_modules/ember-frost-theme/scss'

File: scss/themes/_default.scss [hunk 2]
```diff
@@ -1,20 +1,58 @@
 @import '../colors';
 
 @mixin frost-theme-default {
+
+  // ================================================================
+  // Colors
+  // ================================================================
+
+  // Global
+  $frost-input: $frost-color-grey-1 !global;
```
Comment (sglanzer-fire) [inline scss/themes/_default.scss]: These should be -color- as well

Comment (job13er) [inline scss/themes/_default.scss]: good catch

File: README.md (hunk 1 above)
Comment (job13er) [inline README.md:30]: so should I pull it out?

**Comment by sglanzer-fire** [comment]
Approved - we need to update pullapprove to include more people here

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-theme/pull-request/19/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-theme)

File: README.md (hunk 1 above)
Comment (sglanzer-fire) [inline README.md:30]: I think we have to for now

**Comment by job13er** [comment]
Sorry, @sglanzer gonna have to approve it again, I fixed the `-color-` variables, and yes, I wasn't sure who else it needs, the `ux` template in pullapprove is currently you and I

**Comment by sglanzer-fire** [comment]
I think it stays approved - doesn't look like pullapprove unapproves after a change

File: README.md (hunk 1 above)
Comment (job13er) [inline README.md:30]: Can you update it with the _correct_ way, since I haven't consumed it yet? (in another PR)

**Pull request merged**